core/src/services/s3/error.rs

```diff
@@ -79,6 +71,34 @@ pub async fn parse_error(resp: Response<IncomingAsyncBody>) -> Result<Error> {
     Ok(err)
 }
 
+/// Returns the Errorkind of this code and whether the error is retryable.
+/// All possible error code: <https://docs.aws.amazon.com/AmazonS3/latest/API/ErrorResponses.html#ErrorCodeList>
+pub fn kind_and_retryable(code: &str) -> Option<(ErrorKind, bool)> {
```

Comment:
I prefer to have an API like:

`fn parse_s3_error_code(code: &str) -> (ErrorKind, bool)`

Reply:
Nice one, the naming I chose was bad :(((((